admin_manual/maintenance/upgrade.rst

```diff
@@ -2,93 +2,48 @@
-..   section is the Updater app.*
-   
-.. The core update notifier has only one function, and that is to display a 
-.. notification when a new ownCloud release is available. Then you decide which 
-.. upgrade method to use. When you maintain your ownCloud server via your Linux 
-.. package manager you should ensure that the Updater app is disabled.
+Before beginning an upgrade, please keep the following points in mind:
+
+- Review :doc:`the release notes </release_notes>` for important information about the needed migration steps during that upgrade to help ensure a smooth upgrade process.
+- Skipping major releases is not supported. However *you can* migrate from 8.2 or 9.0 straight to 10.0.   
+- Downgrading is not supported.
+- Upgrading is disruptive, as your ownCloud server will be put into :ref:`maintenance mode <maintenance_mode_label>`. 
+- Large installations may take several hours to complete the upgrade.
+- Downgrading **is not supported** as it risks corrupting your data. If you want to revert to an older ownCloud version, make a new, fresh installation and then restore your data from backup. Before doing this, file a support ticket (if you have paid support) or ask for help in the ownCloud forums to resolve your issue without downgrading.
   
 Prerequisites
 -------------
 
-You should always maintain :doc:`regular backups <backup>` and make a fresh 
-backup before every upgrade.
+We strongly recommend that you always maintain :doc:`regular backups <backup>` as well as make a fresh backup before every upgrade.
+We also recommend that you review any installed third-party apps for compatibility with the new ownCloud release. 
+Ensure that they are all disabled before beginning the upgrade.
+After the upgrade is complete re-enable any which are compatible with the new release.
 
-Then review third-party apps, if you have any, for compatibility with the new 
-ownCloud release. Any apps that are not developed by ownCloud show a 3rd party 
-designation. **Install unsupported apps at your own risk**. Then, before the 
-upgrade, all 3rd party apps must be disabled. After the upgrade is complete you 
-may re-enable them.
+.. warning::
+   Install unsupported apps at your own risk.
 
 .. _owncloud.org/install/:
    https://owncloud.org/install/  
 
-Previous ownCloud Releases
---------------------------
+Upgrade Options
+---------------
+
+There are three ways to upgrade your ownCloud server:
+
+1. Using your :doc:`Linux package manager <package_upgrade>` with our official ownCloud repositories. This is the recommended method. 
+2. Using the :doc:`Updater App <update>` (Server Edition only). This is recommended for shared hosters and for users who want an easy way to track different release channels. It is *not available* and *not supported* on the Enterprise edition.
+3. :doc:`Manually upgrading <manual_upgrade>` with the latest ownCloud release from `owncloud.org/install/`_. 
```

Comment:
In my opinion, we should rather advice for the manual upgrade process as the recommended method